minimesos/src/main/java/com/containersol/minimesos/cluster/ClusterMember.java

```diff
@@ -0,0 +1,42 @@
+package com.containersol.minimesos.cluster;
+
+/**
+ * Generic functionality of every cluster member
+ */
+public interface ClusterMember {
+
+    MesosCluster getCluster();
+    void setCluster(MesosCluster mesosCluster);
+
+    /**
+     * @return the IP address of the container
+     */
+    String getIpAddress();
+
+    /**
+     * Builds container name following the naming convention
+     *
+     * @return container name
+     */
+    String getName();
+
+    /**
+     * @return the ID of the container.
+     */
+    String getContainerId();
+
+    /**
+     * Starts the container and waits until is started
+     *
+     * @param timeout in seconds
+     */
+    void start(int timeout);
+
+    String getRole();
+
+    /**
+     * Removes a container with force
+     */
+    void remove();
+
+}
```

Comment:
Ok cool. Have a look at some of the identifiers. They are still use `container` instead of `process`